AppAuth-JS/built/token_response.d.ts

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork